### PR TITLE
Make Ubuntu 18.04 preferred version

### PR DESCRIPTION
**Pull request by hamishwillee**

### CODE REVIEW

File: en/setup/config_initial.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ The equipment below is highly recommended:
 
 * A Taranis Plus remote control for the safety pilot (or equivalent)
 * A development computer:
-  * MacBook Pro (early 2015 and later) with OSX 10.13 or later
-  * Lenovo Thinkpad 450 (i5) with Ubuntu Linux 16.04 or later
+  * MacBook Pro (early 2015 and later) with OSX 10.13 or later
```
Comment (hamishwillee) [inline en/setup/config_initial.md]: @julianoes Does any of the rest of this need to be changed?

**Review by MaEtUgR — APPROVED** [review]

File: en/setup/config_initial.md (hunk 1 above)
Comment (LorenzMeier) [reply en/setup/config_initial.md]: Should read OS X 10.15 or later

File: en/setup/vscode.md [hunk 2]
```diff
@@ -15,6 +12,9 @@ This topic explains how to setup the IDE and start developing.
 > **Note** There are other powerful IDEs, but they typically take more effort to integrate with PX4.
   With *VScode*, configuration is stored in the PX4/Firmware tree ([Firmware/.vscode](https://github.com/PX4/Firmware/tree/master/.vscode)) so the setup process is as simple as adding the project folder.
 
+<span></span>
+> **Tip** On Ubuntu 16.04, VSCode can be used to build PX4 but not debug targets.
```
Comment (LorenzMeier) [inline en/setup/vscode.md]: Could you add: [...] We therefore recommend using Ubuntu 18.04 instead. (or just deleting the comment). The way its put right now it sounds like 16.04 is the supported target but has deficiencies.

Comment (hamishwillee) [reply en/setup/vscode.md]: Yes, decided to remove. We already suggest that this works on 18.04 above.

**Pull request merged**